api/docgen/cmd/docgen.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"encoding/json"
+	"log"
+	"os"
+
+	"github.com/celestiaorg/celestia-node/api/docgen"
+	"github.com/celestiaorg/celestia-node/nodebuilder"
+)
+
+func main() {
```

Comment:
Should we move it to the central `cmd` instead? It can easily import API pkg. Feels weird to me to have it here

Reply:
done, and added in make file